src/index.js

```diff
@@ -560,6 +615,18 @@ function mergeManifests(manifest1, manifest2) {
   return mergedManifest;
 }
 
+function findAndMap(filePaths, fileContents) {
+  return filePaths.map((filePath) => {
+    const fileContent = fileContents.filter(
+      (fileContent) => fileContent.filePath === filePath
+    )[0];
```

Comment:
It seems like [`find`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) fits better here.